src/s3/explorer/s3BucketNode.ts

```diff
@@ -107,7 +111,11 @@ export class S3BucketNode extends AWSTreeNodeBase implements AWSResourceNode, Lo
         return ''
     }
 
-    public toString(): string {
+    public [inspect.custom](): string {
```

Comment:
Never seen this syntax before, what's this do?

Reply:
Overrides the output of `log.info('%O', node)` (See #1108)

Makes it look nicer in the logs. Without this, `%O` would show the full object similar to `JSON.stringify`. Similar to adding a dedicated field called `description` or something except you don't need to explicitly log that field.

Reply:
Oh and the syntax is to define the method name with a name that is a Symbol.
https://javascript.info/symbol

Basically a way to declare you're implementing this interface without ambiguity (accidentally creating a method with the same name).

Reply:
TIL - they're almost the opposite of a Scala [Symbol](https://www.scala-lang.org/api/current/scala/Symbol.html)